src/reducers/src/map-style-updaters.ts

```diff
@@ -90,7 +90,7 @@ export type MapStyle = {
 
 const getDefaultState = (): MapStyle => {
   const visibleLayerGroups = {};
-  const styleType = 'dark';
+  const styleType = 'dark-matter';
```

Comment:
should we define a constant in default settings file with this value.

it will be easier to share the same value across files